packages/manager/modules/netapp/src/dashboard/volumes/template.html

```diff
@@ -54,7 +54,15 @@
             title=":: 'netapp_volumes_mount_path' | translate"
             property="mountPath"
         >
-            <oui-clipboard name="mountPath" model="$value"></oui-clipboard>
+            <oui-clipboard
+                data-ng-if="$row.mountPath"
+                name="mountPath"
+                model="$value"
+            ></oui-clipboard>
+            <i
+                data-ng-if="!$row.mountPath"
+                translate="netapp_volumes_mount_path_not_configurated"
+            ></i>
```

Comment:
```suggestion
            <span
                data-ng-if="!$row.mountPath"
                translate="netapp_volumes_mount_path_not_configurated"
            ></span>
```

Reply:
Text should be in italic this is why I used <i> tag